### PR TITLE
Refactor makefile - automate poc

### PR DESCRIPTION
**Pull request by omertuc**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,27 +1,22 @@
...
-- Monitor the progress using `make ssh` and `journalctl -f -u bootkube.service` or `kubectl --kubeconfig ./mydir/auth/kubeconfig get clusterversion`
+# How to run
+- Set PULL_SECRET environment variable to your pull secret
+- `make start-iso` - Spins up a VM with the the liveCD. This will automatically perform the following actions:
+	- Clone the installer repo with the boostrap-in-place branch
+	- Apply the patches in the patches directory to that repo
+	- Build the openshift installer
+	- Generate the install-config.yaml 
+	- Download the ISO
+	- Patch the ISO with the bootstrap-in-place ignition generated by the patched openshift installer
```
Comment (eranco74) [inline README.md]:
```suggestion
	- Generate the install-config.yaml 
	- Execute the openshift-installer `create single-node-ignition-config` command to generate the bootstrap-in-place-for-live-iso.ign.
	- Download the RHCOS live ISO
	- Embed the bootstrap-in-place Ignition to the ISO.
```

File: ssh/key [hunk 2]
```diff
@@ -0,0 +1,39 @@
+-----BEGIN OPENSSH PRIVATE KEY-----
```
Comment (markmc) [inline ssh/key:1]: Was this committed in error?

Comment (omertuc) [reply ssh/key:1]: It's a (hopefully) dummy SSH key generated by @eranco74, I've taken it from the BiP SNO PoC branch in the installer repo

Comment (eranco74) [reply ssh/key:1]: No worries, It's not my private key.

Comment (markmc) [reply ssh/key:1]: I didn't check it into this repo for a reason - there should never be a need to check a private SSH key into git. Even if you think it's safe, it's almost sure to cause lots of people in futue to wonder whether it's a leak

Comment (omertuc) [reply ssh/key:1]: https://github.com/eranco74/bootstrap-in-place-poc/pull/4

**Pull request merged**